doc/guide.rst

```diff
@@ -750,3 +750,53 @@ an input to the above mentioned genotyping commands. This can be done by running
     whatshap find_snv_candidates ref.fasta input.bam -o variants.vcf
 
 If Nanopore reads are used for calling SNPs, it is recommended to add option --nanopore to the above command.
+
+Polyploid Phasing
+=================
+
+In addition to diploid phasing, WhatsHap also supports polyploid phasing 
+through a different algorithm. The ``whatshap polyphase`` command works 
+almost the same as the ``phase`` command with a few restrictions:
+
+1. An additional integer argument ``--ploidy`` must be specified. This ploidy
+must match the ploidy in the provided VCF file(s). The ploidy also greatly
+impacts the running time as the phasing becomes more complex. Ploidies
+higher than 6 may take very long to process.
+
+2. WhatsHap will use available genotype information from the VCF file(s), but
+the computed haplotypes are not guaranteed to follow these genotypes, if they
+deviate too much from the allele distribution among the aligned reads.
+
+3. Polyploid phasing on pedigrees is not supported yet.
+
+4. Structural variants are not considered during the phasing. The algorithm
+will always produce the provided number of haplotypes at any location.
+
```

Comment:
What does structural variant mean hear? Does it do indels?

Reply:
I think we never tested indels, but only SNPs. Structural variant means long indels in which not all haplotypes are present. The threading will always output --ploidy many haplotypes at all times. For us this is clear, but users might not expect it. I can  write this part in more detail.

Reply:
Ok, how about this to clarify:

```suggestion

4. The phasing algorithm does not consider copy number variants and always produces the provided number of haplotypes at any location.

```

Reply:
I'd use the term copy number variants instead of "long indels" or "structural variants" here...

Reply:
I'm fine with it. Changed now.